src/stylesheets/components/_timeline.scss

```diff
@@ -8,7 +8,7 @@
     content: '';
     background-color: $grey-1;
     width: 1px;
-    height: 100%;
+    height: calc(100% - 20px);
```

Comment:
Is 20px fixed?

Reply:
yes

Reply:
Should change the value to 21px to cater for one zoom size up and down.